src/3rd_party-static/jsoncpp/src/lib_json/json_reader.cpp

```diff
@@ -140,8 +141,9 @@ Reader::parse( const char *beginDoc, const char *endDoc,
    while ( !nodes_.empty() )
       nodes_.pop();
    nodes_.push( &root );
-   
-   bool successful = readValue();
+
+   int depth_level = 0;
```

Comment:
@ZhdanovP What the reason to have depth_level? As I see you don't use it.
Why depth_level variable sets up inside readValue and this value returns to called function, I believe that adjusting callable function from outside more clear way

Reply:
@KVGrygoriev We have added `depth_level` for control the depth of recursion during parsing. 
I'am not sure that we need to move this parameter outside because it's internal logic of Reader::parse